app/helpers/ingest_job_helper.rb

```diff
@@ -155,6 +158,9 @@ def ingest!(ingest_request, test_mode = false)
 
       Delayed::Worker.logger.info("ingested capture #{uuid}", uuid: ingest_request.uuid)
     end
+    
+    # sometimes captures are deleted or suppressed, and we need to pull them back
+    repo_solr.delete_unseen_captures_below(ingest_request.uuid, seen_capture_uuids)
 
     repo_solr.commit_index_changes
```

Comment:
I am wondering about this commit now that line 163 might be making commits. It gets a little confusing, but it looks like they might be referencing the same @rsolr instance. Should we be committing the prior changes first, or do we want to make sure we can also delete the suppressed/deleted captures before doing so?

Reply:
I went a little back and forth on this. They are referencing the same solr. I think I feel it's ok to delete removed captures as they are probably very sensitive items, even if the update for the parent item doesn't work. I think we want those deletes to happen no matter what. 

But I'm truly on the fence.

Reply:
It just seemed odd that the deletion method would be committing changes that happened previous to it. I don't _think_ it's a big deal but was slightly concerned if that bucket was already pretty full and then we add a bunch of stuff to it.

Reply:
you have a point. I'll switch things up.